radix-transactions/src/model/versioned.rs

```diff
@@ -36,30 +36,14 @@ const V1_FLASH_TRANSACTION: u8 = 8;
 /// These represent the different transaction types.
 #[derive(Clone, Debug, Eq, PartialEq, ManifestSbor)]
 pub enum VersionedTransactionPayload {
-    #[sbor(discriminator(V1_INTENT))]
-    IntentV1 {
-        header: TransactionHeaderV1,
-        instructions: InstructionsV1,
-        blobs: BlobsV1,
-        message: MessageV1,
-    },
-    #[sbor(discriminator(V1_SIGNED_INTENT))]
-    SignedIntentV1 {
-        intent: IntentV1,
-        intent_signatures: IntentSignaturesV1,
-    },
-    #[sbor(discriminator(V1_NOTARIZED_TRANSACTION))]
-    NotarizedTransactionV1 {
-        signed_intent: SignedIntentV1,
-        notary_signature: NotarySignatureV1,
-    },
-    #[sbor(discriminator(V1_SYSTEM_TRANSACTION))]
-    SystemTransactionV1 {
-        instructions: InstructionsV1,
-        blobs: BlobsV1,
-        pre_allocated_addresses: Vec<PreAllocatedAddress>,
-        hash_for_execution: Hash,
-    },
+    #[sbor(flatten, discriminator(V1_INTENT))]
```

Comment:
I guess the above TODO is no longer needed.

Reply:
Good catch! :)